frontend/ui/widget/dictquicklookup.lua

```diff
@@ -156,6 +158,15 @@ function DictQuickLookup:isDocless()
     return self.ui == nil or self.ui.highlight == nil
 end
 
+function DictQuickLookup:getHtmlDictionaryCss()
+    -- Using Noto Sans becaue Nimbus doesn't contain the IPA symbols.
```

Comment:
Typo. Also, IPA is indeed quite important! :-D

Reply:
Nice catch. :)